### ISSUE TITLE
Feature proposal: Disabling stylelint for specific language identifiers (e.g. javascriptreact, javascript)

### ISSUE THREAD

**Issue by hinok** [shinnn/vscode-stylelint#226]
I'd like to disable `vscode-stylelint` for `javascriptreact`, `javascript` because at the moment it reports things that shouldn't be reported (in my opinion) like:

```javascript
render() {
  const styles = {}; //    <---- here is reported error by stylelint, look at the screen below
  
  if (something) {
    // it's just example
    styles.border = '1px solid teal';
  }
}
```

![zrzut ekranu 2018-06-27 o 16 04 50](https://user-images.githubusercontent.com/1313605/41979265-f6d16a54-7a23-11e8-94a2-f3471baa6a72.png)

I'm using already eslint and prettier for linting / fixing js files, stylelint for me is too much.
What do you think?

**Comment by hinok** [comment]
Ooops, sorry for adding duplicate https://github.com/shinnn/vscode-stylelint/issues/173
